contracts/factory/src/tests/config.rs

```diff
@@ -107,6 +113,92 @@ fn factory_successfully_inits_lp() {
     );
 }
 
+#[test]
+fn factory_successfully_inits_stable_pool() {
+    let env = Env::default();
+    env.mock_all_auths();
+    env.budget().reset_unlimited();
+
+    let admin = Address::generate(&env);
+    let token1_admin = Address::generate(&env);
+    let token2_admin = Address::generate(&env);
+    let user = Address::generate(&env);
+
+    let mut token1 = install_and_deploy_token_contract(
+        &env,
+        &token1_admin.clone(),
+        &7,
+        &String::from_str(&env, "Phoenix"),
+        &String::from_str(&env, "PHO"),
+    );
+    let mut token2 = install_and_deploy_token_contract(
+        &env,
+        &token2_admin.clone(),
+        &7,
+        &String::from_str(&env, "USD Coin"),
+        &String::from_str(&env, "USDC"),
+    );
+
+    if token2.address < token1.address {
+        std::mem::swap(&mut token1, &mut token2);
+    }
+
+    let factory = deploy_factory_contract(&env, Some(admin.clone()));
+    assert_eq!(factory.get_admin(), admin);
+
+    let token_init_info = TokenInitInfo {
+        token_a: token1.address,
+        token_b: token2.address,
+    };
+    let stake_init_info = StakeInitInfo {
+        min_bond: 10i128,
+        min_reward: 5i128,
+        manager: Address::generate(&env),
+        max_complexity: 10u32,
+    };
+
+    let lp_init_info = LiquidityPoolInitInfo {
+        admin: admin.clone(),
+        fee_recipient: user.clone(),
+        max_allowed_slippage_bps: 5_000,
+        max_allowed_spread_bps: 500,
+        swap_fee_bps: 0,
+        max_referral_bps: 5_000,
+        token_init_info: token_init_info.clone(),
+        stake_init_info,
+    };
```

Comment:
Didn't we have some helper for that, or do I remember it from somewhere else?

Reply:
we used a helper `generate_lp_init_info` from setup, reduced the code with it a bit